rebar.lock

```diff
@@ -1,16 +1,24 @@
 {"1.1.0",
 [{<<"base16">>,{pkg,<<"base16">>,<<"1.0.0">>},1},
+ {<<"dogstatsd">>,
```

Comment:
we should probably kill the lock file at the library/app level here.

Reply:
ok, let's do this